apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -1495,6 +1495,40 @@ func (rpl *ResourcePlacementList) GetPlacementObjs() []PlacementObj {
 	return objs
 }
 
+// +genclient
+// +genclient:Namespaced
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:scope="Namespaced"
+// +kubebuilder:subresource:status
+// +kubebuilder:storageversion
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ManagedNamespacePlacement is used to represent which clusters the namespace has been placed on to.
+type ManagedNamespacePlacement struct {
```

Comment:
How do you know which CRP this status is for? Can one namespace selected/placed by N CRPs?